sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateClient.java

```diff
@@ -904,8 +859,8 @@ public CertificateIssuer deleteIssuer(String name) {
      * @return A {@link PagedIterable} containing all of the {@link IssuerProperties certificate issuers} in the vault.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public PagedIterable<IssuerProperties> listIssuers() {
-        return listIssuers(Context.NONE);
+    public PagedIterable<IssuerProperties> listPropertiesOfIssuers() {
```

Comment:
Need to make this change in the code snippet tag

Reply:
done